examples/hello/package.json

```diff
@@ -3,8 +3,14 @@
   "description": "An example app demonstrating how Oy might be used.",
   "version": "0.0.1",
   "private": true,
+  "scripts": {
+    "start": "babel-node --stage es2015,stage-2 server.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
   "dependencies": {
     "babel": "^5.8.3",
+    "babel-preset-es2015": "^6.5.0",
+    "babel-preset-stage-2": "^6.5.0",
```

Comment:
Is this actually necessary? I don't think we use any syntax from stage 2 or 3.

https://babeljs.io/docs/plugins/preset-stage-2/